### DISCUSSION TITLE
Everything service is not running

### DISCUSSION THREAD

**Discussion by razikuljoni** [Flow-Launcher/Flow.Launcher#1858] [Q&A]
#solved
everything is not working. In file explorer it is showing that "Warning: Everything service is not running". Flow Launcher version 1.12.1.
![image](https://user-images.githubusercontent.com/68385662/215309933-602a6e6e-ecb1-482d-9d5b-2a470d45f38a.png)

**Comment by claudiolorcajr** [comment]
![Captura de tela 2023-02-03 130619](https://user-images.githubusercontent.com/65817109/216650670-8ae2e955-7548-40ad-969b-67e6d63f9734.png)
I've tried to install some versions, but this error always occurs. Follow the print
Windows 11 - i7 8700k 16GB RAM
tks, regards.
claudio

**Reply by z1nc0r3** [reply]
The usual FL installation folder is **user/Appdata**. Please try installing in the default location or **Program Files** or **Program Files (x86)** folders.
